src/Trustline.ts

```diff
@@ -3,119 +3,205 @@ import { Utils } from './Utils'
 import { User } from './User'
 import { Transaction } from './Transaction'
 import { CurrencyNetwork } from './CurrencyNetwork'
-import { TLOptions } from './typings'
+import {
+  TLOptions,
+  EventFilterOptions,
+  TLEvent,
+  TxObject,
+  TrustlineObject,
+  TrustlineRaw
+} from './typings'
 
+/**
+ * The Trustline class contains all relevant methods for retrieving, creating and
+ * editing trustlines.
+ */
 export class Trustline {
+  private _event: Event
+  private _user: User
+  private _utils: Utils
+  private _transaction: Transaction
+  private _currencyNetwork: CurrencyNetwork
 
-  constructor (private event: Event,
-               private user: User,
-               private utils: Utils,
-               private transaction: Transaction,
-               private currencyNetwork: CurrencyNetwork) {
+  constructor (
+    event: Event,
+    user: User,
+    utils: Utils,
+    transaction: Transaction,
+    currencyNetwork: CurrencyNetwork
+  ) {
+    this._event = event
+    this._user = user
+    this._utils = utils
+    this._transaction = transaction
+    this._currencyNetwork = currencyNetwork
   }
 
   /**
-   * Prepares a raw transaction for a trustline update request
-   * @param network address of currency network
-   * @param debtor address of counterparty who receives trustline update request
-   * @param creditLineGiven value of credit line given to counterparty, i.e. 1.23 if network has to 2 decimals
-   * @param creditLineReceived value of credit line received by counterparty, i.e. 1.23 if network has to 2 decimals
-   * @param decimals (optional) number of decimals can be provided manually
-   * @param gasLimit (optional)
-   * @param gasPrice (optional)
+   * Prepares a tx object for creating a trustline update request. Called by initiator
+   * of update request.
+   * @param network Address of a currency network.
+   * @param counterparty Address of counterparty who receives trustline update request.
+   * @param given Proposed creditline limit given by iniator to counterparty,
```

Comment:
creditlineGiven to make it more clear already from the parameter name

Reply:
https://github.com/trustlines-network/clientlib/issues/87